eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Certs.hs

```diff
@@ -137,20 +161,54 @@ instance
 
   transitionRules = [conwayCertsTransition @era]
 
+-- | CERT-base :
+--     let open PParams pp; open CertState st; open GState gState
+--         refresh = mapPartial (λ v → let open GovVote v in case role of λ where
+--           GovRole.DRep → just credential
+--           _    → nothing) (fromList vs)
+--     in ⊤ -- TODO: check that the withdrawals are correct here
+--     ────────────────────────────────
+--     ⟦ e , pp , vs ⟧ᶜ ⊢ st ⇀⦇ _ ,CERTBASE⦈ record st { gState = record gState
+--                          { dreps = constMap refresh (e +ᵉ' drepActivity) ∪ᵐˡ dreps } }
```

Comment:
My thinking behind doing this was that:
1. We would keep the version of spec that matched the code together
2. When the spec changes, we can diff the version documented in haskell comments and more easily understand what exactly changed in the spec, compared to the implementation
3. Update the spec comments in Haskell along with updating the implementation, this way comments and code in Haskell always correspond to each other.

I will remove it now though, since there is something I'm sure I'm not thinking of. :+1: cc. @Soupstraw